openssl/src/crypto/rsa.rs

```diff
@@ -0,0 +1,39 @@
+use ffi;
+use bn::BigNum;
+use std::fmt;
+
+pub struct RSA {
+    pub rsa_obj : ffi::RSA
```

Comment:
I'd prefer this to be private, and methods to be added to pull the relevant fields.